core/src/main/java/nl/nn/adapterframework/webcontrol/api/TransactionalStorage.java

```diff
@@ -67,14 +73,16 @@
 public class TransactionalStorage extends Base {
 
 	protected static final TransactionDefinition TXNEW = new DefaultTransactionDefinition(TransactionDefinition.PROPAGATION_REQUIRES_NEW);
+	private static final String STORAGESOURCE_TYPE_PIPE = "pipes";
 
 	@GET
 	@RolesAllowed({"IbisDataAdmin", "IbisAdmin", "IbisTester"})
-	@Path("/adapters/{adapterName}/receivers/{receiverName}/stores/{processState}/messages/{messageId}")
+	@Path("/adapters/{adapterName}/{storageSource}/{storageSourceName}/stores/{processState}/messages/{messageId}")
```

Comment:
storagesource can be one of two things correct?
receivers or pipes
Perhaps you can turn this into an enum?
``` 
public enum StorageSource {
		RECEIVERS,PIPES;

		public static StorageSource fromString(String value) {
			return EnumUtils.parse(StorageSource.class, value);
		}
	}
```